### PR TITLE
Fixes issue #1 - blog on moving to github

### PR DESCRIPTION
**Pull request by binnur**
Fixes issue #1 
Also added the Gemfile.lock to .gitignore
Note: while the blog is ready for review, the site needs updates to
support it.

### CODE REVIEW

**Comment by binnur** [comment]
@riyadth please take a look at the new post -- please don't merge it, yet. Thank you!

File: _posts/2019-07-12-moved.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+author_avatar: binnur
+show_avatar : true
+read_time : 3
+feature_image: feature-fish2
+show_related_posts: false
+square_related: recommend-mountain
+---
+Welcome to my new site! This is a behind the scenes look at my move off of
+WordPress. There are two significant aspects to this move:
+1. I am dusting off my blog with goal to write more
```
Comment (riyadth) [inline _posts/2019-07-12-moved.md]: 'with goal' should be 'with the goal'

File: _posts/2019-07-12-moved.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+2. After years of using WordPress, change is good, really good!!
+
+## So, why GitHub, and why now?
+> "There is virtue in work and there is virtue in rest. Use both and overlook
+neither."
+> Alan Cohen
+
+I have been enjoying my personal break and dusting off some of my old habits, which
```
Comment (riyadth) [inline _posts/2019-07-12-moved.md]: 'dusting off' should be 'bringing back'

File: _posts/2019-07-12-moved.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+I miss... As I reflected on blogging, I realized I had a love/hate relationship
+with my old blog. I enjoyed verbalizing my thoughts, but it
+also felt too heavy and time consuming. Part of it was how I approached to my
```
Comment (riyadth) [inline _posts/2019-07-12-moved.md]: 'approached to my' should be 'approached my'

File: _posts/2019-07-12-moved.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+1. Create a light-weight writing experience where I can experiment easily
+2. Simplify the publishing platform and make it fun!
+
+I have used WordPress for a long time, even hosted it myself in the early days.
+However, as an engineer at heart, I like the simplicity of git, how it handles
+versioning, and I have been looking for ways to spend more time using it. There is something deeply
+satisfying about using vim for editing, and git for check-ins.
+
+I did try to experiment with Medium for writing -- but, I didn't find it
+touching my creative side. As I type this, I realize I am using VSCode with vim
```
Comment (riyadth) [inline _posts/2019-07-12-moved.md]: 'touching' should be 'touched'

File: _posts/2019-07-12-moved.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+plug-in, which part of this writing experience speaks creative?! And yes, I did install
```
Comment (riyadth) [inline _posts/2019-07-12-moved.md]: I don't understand this sentence: 'As I type this, I realize I am using VSCode with vim plug-in, which part of this writing experience speaks creative?!'
Do you mean that you are taking a creative approach to the process of writing? Saying something like 'using non-standard tools like VSCode and vim enhances my experience, and lets me offset the limitations of the tools with my creativity' or something like that?

File: _posts/2019-07-12-moved.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+a spellchecker extension to VSCode :) Regardless, I
+just couldn't make it pass my rough draft on Medium. I am also not a fan of
```
Comment (riyadth) [inline _posts/2019-07-12-moved.md]: Should 'make it pass' be 'make it past'? Not sure what that sentence means.

File: _posts/2019-07-12-moved.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+Medium's subscription model, so that didn't help.
+
+## Summary: My Workflow
+[Jekyll for Docker](https://hub.docker.com/r/jekyll/jekyll/) made it really easy
+to get started with Jekyll. That coupled with the simplicity of the [Writer
+Jekyll](https://themeforest.net/item/writer-a-minimal-blog-for-jekyll/10562560)
+theme by [AdventureThemes](http://adventurethemes.com/) made it a breeze to get
+started with my new blog site. Setting up a GitHub Page is easy -- just takes a minute. I spent a few hours moving my old content to the
```
Comment (riyadth) [inline _posts/2019-07-12-moved.md:51]: Long line of text, needs newline before 'minute'

File: _posts/2019-07-12-moved.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+new site, and here we go!
+
+With that, my workflow:
+* Start a new blog post on a local branch
+* Edit/preview on vscode
+* Test on a local server via docker-compose
+* Push branch to github and ping my editor, i.e. hubby :) for a quick read
+* Once approved, PR and merge
```
Comment (riyadth) [inline _posts/2019-07-12-moved.md]: Insert line above:
* Fix my syntax errors

File: _posts/2019-07-12-moved.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+
+I also started a GitHub project to capture blog ideas, site improvements, etc.
+It is configured for automated workflow, because why not?! As a bonus, this
+activity got me reinterested in photography, and I have been thinking about how to use
+Apple's Photo's app in my workflow, as Aperture have been discontinued and I haven't found a
```
Comment (riyadth) [inline _posts/2019-07-12-moved.md]: "Photo's" should be "Photos"

File: _posts/2019-07-12-moved.md [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+replacement.
+
+In the process of setting up this blog, I decided to turn off comments. Any
+potential conversations can take place on other social platforms, likely
+twitter. And, at least for now, I am happy with GitHub's Traffic Insights.
```
Comment (riyadth) [inline _posts/2019-07-12-moved.md]: 'twitter' should be 'Twitter'

**Review by riyadth — COMMENTED** [review]
Good, with a few tweaks.

File: _posts/2019-07-12-moved.md (hunk 7 above)
Comment (binnur) [reply _posts/2019-07-12-moved.md:51]: I'll see if there is a good process for handling md files for reviews. Preview works better.

**Comment by binnur** [comment]
Verified blog entry w/ category projects update -- all looks good!

**Comment by binnur** [comment]
Fixes issue #1

**Pull request merged**